Modules/Tokens/src/Tokens.Infrastructure.Database.Postgres/Migrations/20240807121033_PersonalizedTokens.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Backbone.Modules.Tokens.Infrastructure.Database.Postgres.Migrations
+{
+    /// <inheritdoc />
+    public partial class PersonalizedTokens : Migration
```

Comment:
Shouldn't we have a `ForeignKey` for the recently added `ForIdentity` column?

Reply:
Nope. We don't have an Identities table in the Schemas module.

We don't have any foreign keys to the Identities table for any tables outside of the Devices module